src/components/MovieCard/MovieCard.js

```diff
@@ -1,6 +1,48 @@
 import React from 'react';
 import './MovieCard.scss';
+import PropTypes from 'prop-types';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({
+  title,
+  description,
+  imgUrl,
+  imdbUrl,
+}) => (
+  <div>
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          src={imgUrl}
+          alt="Film logo"
+        />
+      </figure>
+    </div>
+
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src="/images/imdb-logo.jpeg" alt="imdb" />
```

Comment:
`src="images/imdb-logo.jpeg"` makes image visible

Reply:
fix this
![image](https://user-images.githubusercontent.com/57598825/104446004-80dd9400-55a2-11eb-966b-fcfb771f7061.png)